C/tests/c4DatabaseTest.cc

```diff
@@ -1474,3 +1474,61 @@ N_WAY_TEST_CASE_METHOD(C4DatabaseTest, "Re-open database with an index", "[Datab
     REQUIRE(c4coll_deleteIndex(darthColl, "byAnswer"_sl, ERROR_INFO()));
     REQUIRE(c4coll_deleteIndex(darthColl, "byResult"_sl, ERROR_INFO()));
 }
+
+#define BENCH(expr)                                                            \
+  [&]() {                                                                      \
+    auto start = std::chrono::system_clock::now();                             \
+    expr;                                                                      \
+    auto end = std::chrono::system_clock::now();                               \
+    return end - start;                                                        \
+  }()
+
+// CBL-5241 / CBL-5254
+// Before the fix associated with this ticket, c4doc_update performs a SCAN query 
+// when fetching rev-tree, due to the lack of sequence index. This means the update
+// would get increasingly slower for every doc existing in the DB.
+N_WAY_TEST_CASE_METHOD(C4DatabaseTest, "c4doc_update sequence index perf", "[Database][C]") {
```

Comment:
I don't think we should add the perf test in the test here. The perf test should be in c4PerfTest.cc here. 

The fix itself is good enough and you can create another ticket to put the test in c4PerfTest and also showfast as a doc update perferformance test.